addon/components/frost-action-link.js

```diff
@@ -0,0 +1,30 @@
+import Ember from 'ember'
+import layout from '../templates/components/frost-action-link'
+import _frostObjectDetails from './frost-object-details'
+import _ from 'lodash/lodash'
+
+export default Ember.Component.extend({
+  routing: Ember.inject.service('-routing'),
+
+  layout: layout,
+  classNames: ['frost-action-link'],
+  classNameBindings: ['isSelected'],
+
+  initContext: Ember.on('didInitAttrs', function () {
+    this.set('_frostObjectDetails', this.nearestOfType(_frostObjectDetails))
```

Comment:
This isn't considered good practice - can we pass the action directly to the action-link via the template?